creature/Dragonix; Monster Manual Expanded.json

```
@@ -63571,11 +63571,6 @@
 						"The cambion is a 13th-level spellcaster. Its spellcasting ability is Intelligence (spell save {@dc 16}, {@hit 8} to hit with spell attacks). It has the following wizard spells prepared:"
 					],
 					"spells": {
```

There is a cantrip list on my copy (v1.19).
```suggestion
					"spells": {
						"0": {
							"spells": [
								"{@spell infestation|XGE}",
								"{@spell mage hand}",
								"{@spell message}",
								"{@spell poison spray}"
							]
						},
```

---

Ah, I must have an old copy. I've added your suggestion.